portfolio/src/main/webapp/about.html

```diff
@@ -0,0 +1,84 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>sehhaj portfolio</title>
+    <link rel="stylesheet" href="style.css">
+    <script src="script.js"></script>
+  </head>
+  <body>
+    <div id="content">
+      <h1>get to know all about sehhaj</h1>
+      <img src="./images/avatar.png" style="width:50%;height:50%;">
+      <p>hi internet! nice to meet yall &#x1F44B;</p>
```

Comment:
Nit: You should be able to just include the character without using a reference (&#x1F44B;), see https://google.github.io/styleguide/htmlcssguide.html#Entity_References

Reply:
Did not know you could directly include characters. Made the changes